shapely/tests/geometry/test_geometry_base.py

```diff
@@ -83,3 +84,10 @@ def test_GeometryType_deprecated():
         geom_type = geom.geometryType()
 
     assert geom_type == geom.geom_type
+
+
+@pytest.mark.skipif(shapely.geos_version < (3, 10, 0), reason="GEOS < 3.10")
+def test_segmentize():
+    line = LineString([(0, 0), (0, 10)])
+    result = line.segmentize(tolerance=5)
+    assert result.equals(shapely.segmentize(line, tolerance=5))
```

Comment:
> the test result should be independent from the method:

Note that it was (intentionally) comparing to the _function_, not method (itself). Because in my thinking that was sufficient (we basically only want to test that the method uses the function under the hood, and it's the function that is more extensively tested for actual results)

Now, doesn't matter that much, the updated way is fine as well, I just wanted to note that the previous way was still testing something useful ;)